terraform/dnsimple/dns/main.tf

```diff
@@ -9,6 +9,8 @@ variable short_name {}
 variable subdomain { default = "" }
 variable worker_count {}
 variable worker_ips {}
+variable kubeworker_count {}
+variable kubeworker_ips {}
```

Comment:
should these default to `0` and `""` to avoid breaking existing installations, or are we fine with that?